packages/host/app/components/ai-assistant/apply-button/usage.gts

```diff
@@ -45,7 +48,7 @@ export default class AiAssistantApplyButtonUsage extends Component {
         <Args.String
           @name='state'
           @value={{this.state}}
-          @options={{array 'ready' 'applying' 'applied' 'failed'}}
+          @options={{array 'ready' 'applying' 'applied' 'failed' 'preparing'}}
```

Comment:
This is only happening for me in Freestyle so it’s harmless but it’s strange

![screencast 2025-03-10 08-40-10](https://github.com/user-attachments/assets/006fa825-b7ee-469b-86f3-1b423a96583d)